packages/governance/src/contractGovernance/governParam.js

```diff
@@ -21,10 +20,15 @@ const { details: X, quote: q } = assert;
 const CONTRACT_ELECTORATE = 'Electorate';
 
 /** @type {MakeParamChangePositions} */
-const makeParamChangePositions = (paramSpec, proposedValue) => {
-  const positive = harden({ changeParam: paramSpec, proposedValue });
-  const negative = harden({ noChange: paramSpec });
-  return { positive, negative };
+const makeParamChangePositions = changes => {
+  const dense = changes.map(c => {
```

Comment:
`dense` and `sparse` aren't clear to me. Also, why is `dense` an array ob objects with a single name:value ? 

Consider `paramValues` as a mapping of all names to their values.